modules/timepoint_list/test/timepoint_listTest.php

```diff
@@ -1,30 +1,136 @@
 <?php
 /**
- * timepoint_list automated integration tests
+ * Timepoint_list automated integration tests
  *
  * PHP Version 5
  *
  * @category Test
  * @package  Loris
- * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @author   Nicolas Brossard <nicolasbrossard.mni@gmail.com>
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
-
-require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTestWithCandidate.class.inc";
-class timepointListTestIntegrationTest extends LorisIntegrationTestWithCandidate
+require_once __DIR__
+    . "/../../../test/integrationtests/LorisIntegrationTestWithCandidate.class.inc";
+/**
+ * TimepointListIntegrationTest
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Nicolas Brossard <nicolasbrossard.mni@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class TimepointListIntegrationTest extends LorisIntegrationTestWithCandidate
 {
+    private static $_TEST_SESSION = array(
+                                     'Test',
+                                     '',
+                                     '',
+                                     '',
+                                     '',
+                                     '-',
+                                     '',
+                                     '-',
+                                     '',
+                                     '',
+                                     '',
+                                    );
     /**
      * Tests that, when loading the timepoint_list module, some
      * text appears in the body.
      *
      * @return void
      */
-    function testTimepointListDoespageLoad()
+    function testTimepointListPageLoad()
+    {
+        $this->safeGet($this->url . "/timepoint_list/?candID=000000");
```

Comment:
I think there was another pull request last week that changed this CandID. You might want to rebase this PR to make sure it still works..